apis.py

```diff
@@ -18,14 +18,16 @@
 from database_entry import add_requests, add_volunteers_to_db, contact_us_form_add, verify_user, \
     add_user, request_matching, update_requests_db, update_volunteers_db, \
     blacklist_token, send_sms, send_otp, resend_otp, verify_otp, update_nearby_volunteers_db, \
-    add_request_verification_db, update_request_v_db, update_request_status, save_request_sms_url
+    add_request_verification_db, update_request_v_db, update_request_status, save_request_sms_url, \
+    update_request_updates_db
```

Comment:
what is this function for? never saw this before

Reply:
This is an existing function which I am reusing here.

Reply:
you are not using it anywhere

Reply:
removed